### PR TITLE
feat(dedicated.account): validation message clarification in the manager

### PR DESCRIPTION
**Pull request by MaximeBajeux**
Avoid to have an info and a success banner at the same time when an indian user try to upload documents Ref: MANAGER-13110

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-13110
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

I add a condition to avoid the info banner appear at the same time of the success banner when the user upload a document.

## Related

Translations:
- [x] CMT-10584

### CODE REVIEW

**Review by anooparveti — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "user_account_identity_documents_selection_file_format_invalid": "Format de fichier non pris en charge. Veuillez utiliser l'un des formats acceptés : jpg, jpeg, pdf et png",
   "user_account_identity_documents_submit": "Envoyer mes documents",
   "user_account_identity_documents_submit_success_message_title": "Merci, nous avons bien reçu vos documents!",
-  "user_account_identity_documents_submit_success_message_description": "Nous avons bien reçu l’ensemble de vos documents. Ils seront soumis à validation auprès de nos équipes. Vous serez informé par email de leur validation ou vous serez contacté si nous avons besoin d’informations complémentaires.",
+  "user_account_identity_documents_submit_success_message_description": "Ils seront soumis à validation auprès de nos équipes. Vous serez informé par email de leur validation ou vous serez contacté si nous avons besoin d’informations complémentaires.",
```
Comment (dectotam) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json:20]:
```suggestion
  "user_account_identity_documents_submit_success_message_description": "Ils seront soumis à validation auprès de nos équipes. Vous serez informé par email de leur validation ou vous serez contacté si nous avons besoin d'informations complémentaires.",
```

**Review by dectotam — CHANGES_REQUESTED** [review]

**Review by benjaminlepine — DISMISSED** [review]

File: packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
   "user_account_identity_documents_selection_file_format_invalid": "Format de fichier non pris en charge. Veuillez utiliser l'un des formats acceptés : jpg, jpeg, pdf et png",
   "user_account_identity_documents_submit": "Envoyer mes documents",
   "user_account_identity_documents_submit_success_message_title": "Merci, nous avons bien reçu vos documents!",
```
Comment (dectotam) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json:19]: using exclamation point without a previous space is fairly informal

```suggestion
  "user_account_identity_documents_submit_success_message_title": "Merci, nous avons bien reçu vos documents !",
```

File: packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -17,7 +17,7 @@
...
-  "user_account_identity_documents_submit_success_message_description": "Nous avons bien reçu l’ensemble de vos documents. Ils seront soumis à validation auprès de nos équipes. Vous serez informé par email de leur validation ou vous serez contacté si nous avons besoin d’informations complémentaires.",
+  "user_account_identity_documents_submit_success_message_description": "Ils seront soumis à validation auprès de nos équipes. Vous serez informé par email de leur validation ou vous serez contacté si nous avons besoin d'informations complémentaires.",
   "user_account_identity_documents_submit_error_message_description": "Vos documents n’ont pas été correctement envoyés. Veuillez refaire l’upload et l’envoi de vos documents",
```
Comment (dectotam) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json:21]: I know this code isn't directly concerned by this PR, but if you want, I suggest you change apostrophe in order to use the right format in this sentence too. 
```suggestion
  "user_account_identity_documents_submit_error_message_description": "Vos documents n'ont pas été correctement envoyés. Veuillez refaire l'upload et l'envoi de vos documents",
```

**Review by dectotam — CHANGES_REQUESTED** [review]

**Review by dectotam — DISMISSED** [review]

File: packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -16,12 +16,12 @@
   "user_account_identity_documents_selection_file_format": "Format: jpg, jpeg, pdf, png Taille maximale: 10 mo.",
   "user_account_identity_documents_selection_file_format_invalid": "Format de fichier non pris en charge. Veuillez utiliser l'un des formats acceptés : jpg, jpeg, pdf et png",
   "user_account_identity_documents_submit": "Envoyer mes documents",
-  "user_account_identity_documents_submit_success_message_title": "Merci, nous avons bien reçu vos documents!",
-  "user_account_identity_documents_submit_success_message_description": "Nous avons bien reçu l’ensemble de vos documents. Ils seront soumis à validation auprès de nos équipes. Vous serez informé par email de leur validation ou vous serez contacté si nous avons besoin d’informations complémentaires.",
-  "user_account_identity_documents_submit_error_message_description": "Vos documents n’ont pas été correctement envoyés. Veuillez refaire l’upload et l’envoi de vos documents",
+  "user_account_identity_documents_submit_success_message_title": "Merci, nous avons bien reçu vos documents !",
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json:19]: **question:** Is it requested to add this extra space? If not, can you remove it please?

**Review by benjaminlepine — DISMISSED** [review]

File: packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json (hunk 4 above)
Comment (MaximeBajeux) [reply packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json:19]: Accordingly to @dectotam, normally in French it should have a space here.

**Review by anooparveti — DISMISSED** [review]

**Review by antonymarion — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Pull request merged**